eth/gasprice/gasprice_test.go

```diff
@@ -108,6 +108,7 @@ func newTestBackend(t *testing.T, londonBlock *big.Int, pending bool) *testBacke
 	)
 	config.LondonBlock = londonBlock
 	config.ArrowGlacierBlock = londonBlock
+	config.ArchimedesBlock = londonBlock
```

Comment:
Do we need `config.ShanghaiBlock`?

Reply:
no, those tests don't test anything regarding `Shanghai` just fails on [CheckConfigForkOrder](https://github.com/scroll-tech/go-ethereum/blob/805676e22721093515332352eee02ff64b6d8caf/params/config.go#L724)